src/patternfly/components/Wizard/wizard.scss

```diff
@@ -441,7 +441,9 @@
   word-break: break-word;
 
   &.pf-m-no-padding {
```

Comment:
Do you just want to move this into the pf-c-wizard__main-body class? It's okay if it's breaking

Reply:
lol nevermind was just thinking from react perspective, no more breaking for core :D

Reply:
Thanks for the quick turnaround!~